src/Eloquent/Model.php

```diff
@@ -4,16 +4,22 @@
 
 namespace MongoDB\Laravel\Eloquent;
 
+use Brick\Math\BigDecimal;
```

Comment:
I discover that [Laravel requires this lib](https://github.com/laravel/framework/blob/10.x/composer.json#L27), which has [no stable version](https://packagist.org/packages/brick/math) (>1).

Reply:
> This library requires PHP 8.0 or later. For PHP 7.4 compatibility, you can use version 0.10. For PHP 7.1, 7.2 & 7.3, you can use version 0.9.

It seems the package is not using semantic versioning. However, it is used by 1.2m other repos.